src/pages/Potato.tsx

```diff
@@ -0,0 +1,57 @@
+import axios, {AxiosError} from 'axios';
+import { useState } from 'react';
+
+type Food = {
+  fdcId: number;
+  description: string;
+  foodNutrients: Nutrient[];
+};
+
+type Nutrient = {
+  nutrientName: string;
+  value: number;
+  unitName: string;
+};
+
+const Potato = () => {
+  const [foods, setFoods] = useState<Food[]>([]);
+  
+  const fetchInfo = async () => {
+    try {
+      const fetchResult = await axios.get(`http://localhost:8080/api/usda/search`)
```

Comment:
non-blocking: eventually we will want something like this to have a different URL for local and deployed environments.

```suggestion
      const fetchResult = await axios.get(`${process.env.API_BASE_URL}/api/usda/search`)
```